bolt/include/bolt/Core/DebugNames.h

```diff
@@ -68,6 +68,16 @@ class DWARF5AcceleratorTable {
   std::unique_ptr<DebugBufferVector> releaseBuffer() {
     return std::move(FullTableBuffer);
   }
+  /// Adds a DIE that is referenced across CUs.
+  void addCrossCUDie(const DIE *Die) {
+    CrossCUDies.insert({Die->getOffset(), Die});
+  }
+  /// Returns true if the DIE can generate an entry for a cross cu reference.
+  /// This only checks TAGs of a DIE because when this is invoked DIE might not
+  /// be fully constructed.
```

Comment:
Do we have a case where the TAG of a DIE might not be enough to determinate whether we need to add the cross-TU DIE, or is this always safe?

Reply:
SG